src/OSPSuite.R/Api.cs

```diff
@@ -24,9 +24,9 @@ public static class Api
    {
       public static IContainer Container { get; private set; }
 
-      public static void InitializeOnce(ApiConfig apiConfig)
+      public static void InitializeOnce(ApiConfig apiConfig, Action<IContainer> registerFunction = null)
```

Comment:
Can you change the name of these optional args to `registerAction` instead of `registerFunction`